app/components/listing/TableOfContactInfo.tsx

```diff
@@ -34,7 +34,7 @@ export const TableOfContactInfo = ({ service }: { service: Service }) => {
             <tr>
               <th>Website</th>
               <td>
-                <a href={website}>{website}</a>
+                <a target="_blank" rel="noopener noreferrer" href={website}>{website}</a>
```

Comment:
As a bonus to this work, I added target="_blank" attrs to anchor/link elements on our services that go to external sites per a product request from Tammy. I think it makes a lot of sense to open a new tab when clicking to a new site

Reply:
Good point and thanks for sharing these resources! I try adding the `RiExternalLinkFill`. It looked a little odd with links that took up more than one line. I made a note to talk to the designers during the next Wednesday meeting to get an idea of what they think of opening in a new tab or not or if they want to provide a design